### PR TITLE
Hostile animal spawns further from player

### PR DESCRIPTION
**Pull request by PBWiseman**
## Describe your changes

Hostile animal should now spawn further away from player to avoid instant death.

## Issue number and link

#162 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have written any applicable tests
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] This complies with the [product coding guidelines](https://github.com/BIT-Studio-3/Space-Rescue/wiki/Code-Review-Format)

### CODE REVIEW

File: Assets/Scripts/Managers/SpawningManager.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ void Start()
 
         //Setting the amount of objects to be spawned
         hostileCount = PlanetStates.Instance.planetInfo[PlanetStates.Instance.activePlanet].hostileCount;
+        hostileCount = 20;
```
Comment (ChaseBENNC9) [inline Assets/Scripts/Managers/SpawningManager.cs:54]: Is this line still supposed to be here?

**Review by ChaseBENNC9 — CHANGES_REQUESTED** [review]
### Code:  ✅
Code Follows guidelines and has comments where needed.  I had a question about a particular line in `SpawningManager.cs`.
### Functionality ✅
The Hostile animal no longer seems to spawn close to the player and this issue is solved. Tested in both the editor and a build of the game. I have outlined something i noticed with the spawn count of hostile animals.


### Tests ✅
None required.
### Bugs
I believe you have left in a line for testing in the spawning manager setting hostile count to 20. This spawns 20 hostile animals but the planet counter still says 11. Not sure if this was intentional or not?

Other than that everything looks good

**Review by ChaseBENNC9 — APPROVED** [review]
Concerns fully addressed ✅

**Review by 3MKJOE58 — APPROVED** [review]

**Review by erokasyuart — APPROVED** [review]
5 feels like a large enough space on spawn looks good!

**Pull request merged**